rasa_core/training/interactive.py

```diff
@@ -1060,21 +1069,46 @@ def _validate_nlu(intents: List[Text],
 
 def _correct_entities(latest_message: Dict[Text, Any],
                       endpoint: EndpointConfig,
-                      sender_id: Text) -> Dict[Text, Any]:
+                      sender_id: Text) -> [Dict[Text, Any]]:
```

Comment:
this is not a proper type
```suggestion
                      sender_id: Text) -> List[Dict[Text, Any]]:
```